### PR TITLE
tests: fixed failed tests - stabling CI

### PR DESCRIPTION
**Pull request by anabwan**
* fixed get_csv_files function
  - fixed seed test
* fixed clres on conftest - now can modify paths during test run
  - fixed the mxnet checkpoint test

### CODE REVIEW

File: rl_coach/tests/conftest.py [hunk 1]
```diff
@@ -99,10 +115,10 @@ def __init__(self, csv, log, pattern):
 
     # get the stdout for logs results
     log_file_name = 'test_log_{}.txt'.format(p_name)
-    stdout = open(log_file_name, 'w')
+    # stdout = open(log_file_name, 'w')
```
Comment (galleibo-intel) [inline rl_coach/tests/conftest.py]: can this be removed?

File: rl_coach/tests/test_checkpoint.py [hunk 2]
```diff
@@ -91,7 +91,13 @@ def _create_cmd_and_run(flag):
         return p
 
     if framework == "mxnet":
+        # update presetnamw
```
Comment (galleibo-intel) [inline rl_coach/tests/test_checkpoint.py]: comment is not clear

File: rl_coach/tests/utils/test_utils.py [hunk 3]
```diff
@@ -65,20 +65,26 @@ def read_csv_paths(test_path, filename_pattern, read_csv_tries=120,
     """
     csv_paths = []
     tries_counter = 0
-    while not csv_paths or extra_tries > 0:
+
+    if extra_tries > 0:
```
Comment (galleibo-intel) [inline rl_coach/tests/utils/test_utils.py]: If statement seems redundant. instead I would assert on `extra_tries` value, making sure that it is an int, and that it is >= 0. and then just add it to `read_csv_tries`

**Comment by anabwan** [comment]
fixed all comments

**Pull request merged**